modules/openid_connect/app/views/openid_connect/providers/_form.html.erb

```diff
@@ -1,3 +1,11 @@
+<% if @provider.persisted? && @provider.name == 'azure' && @provider.tenant.empty? %>
+  <div class="op-toast -warning">
+    <div class="op-toast--content">
+      <p><%= 'The configured Azure app points to a deprecated API from Azure. Please create a new Azure app to ensure the functionality in future.' %></p>
```

Comment:
🙏

Reply:
🟡 I probably put that into a locale string.

Reply:
done